### ISSUE TITLE
Window shadow rendering broken on Elementary OS, possibly GNOME

### ISSUE THREAD

**Issue by kmandarin-orange** [flathub/com.mojang.Minecraft#60]
The launcher has odd shadow rendering instead of letting the system handle it, making artifacts if the window is moved.

![Screenshot from 2020-07-28 20-42-28](https://user-images.githubusercontent.com/55755379/88746833-14192f00-d113-11ea-9a0a-dcdbcfeee882.png)
![Screenshot from 2020-07-28 20-42-35](https://user-images.githubusercontent.com/55755379/88746836-15e2f280-d113-11ea-883c-995c851191ea.png)

**Comment by AsciiWolf** [comment]
I don't have this issue on my Fedora machines running GNOME.

**Comment by kmandarin-orange** [comment]
Right, changed the title for better accuracy, unless someone else using Gnome has that problem.
